File: code/ClassGetSet.java
class Animal {
  private String name;

  public void setName(String name) {
    this.name = name;
  }

  public String getName() {
    return this.name;
  }
}

class ClassGetSet {
  public static void main(String[] args) {
    Animal animal = new Animal();
    animal.setName("Kelya");
    System.out.println(animal.getName());
  }
}

File: code/ClassGetSet.php
<?php

class AnimalE
{
    private $name;

    public function setName(string $name)
    {
        $this->name = $name;
    }

    public function getName() : string
    {
        return $this->name;
    }
}


$animal = new AnimalE();
$animal->setName("Kelya");
echo $animal->getName();

